app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,19 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self.hummer_level + 4
```

Comment:
Ensure that the `get_rating` method accesses the `_hammer_level` attribute if you change its name to be protected.